### ISSUE TITLE
On Create Order "Call to a member function setRelation() on null "

### ISSUE THREAD

**Issue by Vinya007** [conedevelopment/bazar#100]
- Bazar Version: 0.4.4
- Laravel Version: 8.25.0
- PHP Version: 7.4.0
- Database Driver & Version: MariaDB

### Description:
On create Order on admin Show error
Call to a member function setRelation() on null .

**Comment by szepeviktor** [comment]
Thank you for your report!

Could you point out in which line it occurs?

https://github.com/conedevelopment/bazar/blob/f67a7e7490172a818a22a022b52ed1e057b02d3a/src/Http/Controllers/OrdersController.php#L72-L82

**Comment by Vinya007** [comment]
```php
/**
     * Get all the items of the model.
     *
     * @return \Illuminate\Support\Collection
     */
    public function getItemsAttribute(): Collection
    {
        return $this->products->map(function (Product $product): Item {
            return $product->item
                ->setRelation('product', $product)
                ->setRelation('itemable', $this);
        });
    }
```
Call to a member function setRelation() on null in file C:\wamp64\www\bazar\vendor\conedevelopment\bazar\src\Concerns\InteractsWithItems.php on line 106

**Comment by szepeviktor** [comment]
Thank you. We are a step [closer](https://github.com/conedevelopment/bazar/blob/78d29ee4b2077d2ecc58db5c88ea9adb1cd9219b/src/Concerns/InteractsWithItems.php#L102-L109).

**Comment by iamgergo** [comment]
I fixed it in the [`v0.4.5`](https://github.com/conedevelopment/bazar/releases/tag/v0.4.5).

Thanks! I don't know how did I skip that error...
